docs/FPGA-ACCEL.md

```diff
@@ -14,3 +16,9 @@ Here is a brief summary of the tasks we have done:
 * Resynthesize the POW hardware accelerator for Arrow Sockit board and flash it into the board
 * Integrate the IOTA PoW hardware accelerator into dcurl's implementation interface
 * Test and verify it
+
+For IOTA FPGA-accelerated integration into dcurl, we consider one dcurl maps one FPGA board unlike the GPU-accelerated solution which is considered a multi-GPU scenario. 
+
+Here is a simple summary of integrating FPGA-accelerated solution into dcurl:
+* Do not have a responsibility to multi-thread management as possible
+* Do not implememt multi-threads

```

Comment:
"No multi-thread management inside the FPGA Implementation Context is implemented"